packages/mdc-button/README.md

```diff
@@ -17,7 +17,7 @@ path: /catalog/buttons/
 </div>-->
 
 The MDC Button component is a spec-aligned button component adhering to the
-[Material Design button requirements](https://material.io/guidelines/components/buttons.html).
+[Material Design button requirements](https://material.io/go/design-buttons).
```

Comment:
FYI I'm pretty sure this paragraph was entirely removed in a recently merged PR, so it'll probably conflict upon updating.

Reply:
Updated.